library/core/src/alloc/layout.rs

```diff
@@ -181,10 +181,16 @@ impl Layout {
     ///     - a [slice], then the length of the slice tail must be an initialized
     ///       integer, and the size of the *entire value*
     ///       (dynamic tail length + statically sized prefix) must fit in `isize`.
+    ///       The pointer address plus the size of the entire value must not
```

Comment:
"entire value" might need clarification

Reply:
It's defined in the previous sentence. Is there anything I should add?

Reply:
Oh oops, you are right. Thanks!